### DISCUSSION TITLE
addGeneExpressionMatrix cell name discrepancies

### DISCUSSION THREAD

**Discussion by grimmem1** [GreenleafLab/ArchR#1823] [Questions / Documentation]
I am running into the below error when trying to combine a complex MIX-seq snATAC Archr object with the corresponding snRNA SE object. Long story short, if I know the location of the cell names in disagreement, it seems I could pretty easily transform them to match, but I am not having much luck finding which part of each data structure addGeneExpressionMatrix is using for the matching. Any insights are appreciated.

`<simpleError in .addMatToArrow(mat = matz, ArrowFile = ArrowFile, Group = paste0(matrixName,     "/", chr), binarize = FALSE, addColSums = TRUE, addRowSums = TRUE,     addRowVarsLog2 = TRUE): CellNames in Matrix Group do not Match CellNames in Matrix Being Written!>
`

**Comment by rcorces** [comment]
I do not have enough information to help you. Bugs and errors should be posted in Issues and you should follow the Issue template provided.
